src/Gateway/Home.jsx

```diff
@@ -395,14 +391,14 @@ return (
       <Container>
         <Pattern $background="linear-gradient(264deg, #CFCCF5 0%, #A39CEC 99.35%)" $borderRadius="16px">
           <PatternContent $padding="55px 24px 55px 48px" $mobilePadding="48px 20px">
-            <Grid $gap="24px" $columns="1fr 1fr">
+            <Grid $gap="24px" $mobileGap="48px" $columns="1fr 1fr">
               <Flex $direction="column" $gap="32px">
                 <H1>Welcome</H1>
                 <Text $size="text-xl" $mobileSize="text-l" style={{ maxWidth: "385px" }}>
                   Start building right away with easy to use developer tools.
                 </Text>
               </Flex>
-              <Flex $direction="column" $gap="16px" $mobileGap="48px">
+              <Flex $direction="column" $gap="0" $mobileGap="48px">
```

Comment:
Overall, this changes looks good to me but I'd like to roll back this `$gap="0"` to:

```suggestion
              <Flex $direction="column" $gap="16px" $mobileGap="48px">
``` 
This change is needed to avoid this collision:

<img width="493" alt="Знімок екрана 2024-05-03 о 18 22 58" src="https://github.com/near/near-discovery-components/assets/34593263/744f51f0-86ed-4d05-9f46-f41881d0ae2c">

There is another similar section as an example how this should look like:

<img width="445" alt="Знімок екрана 2024-05-03 о 18 23 07" src="https://github.com/near/near-discovery-components/assets/34593263/ea0b85b8-1276-4f39-b3da-0438b753589a">

cc @gabehamilton